packages/renderer/src/lib/ui/ConnectionStatus.svelte

```diff
@@ -58,5 +58,5 @@ $: statusStyle = statusesStyle.get(status) || {
 };
 </script>
 
-<div aria-label="Connection Status Icon" class="{roundIconStyle} {statusStyle.bgColor}"></div>
-<span aria-label="Connection Status Label" class="{labelStyle} {statusStyle.txtColor}">{statusStyle.label}</span>
+<div aria-label="connection-status-icon" class="{roundIconStyle} {statusStyle.bgColor}"></div>
+<span aria-label="connection-status-label" class="{labelStyle} {statusStyle.txtColor}">{statusStyle.label}</span>
```

Comment:
to me labels are human labels not with `-` no ?

Reply:
This is the state before, I am reverting it in order to do not mix adding - refactoring unless we agree on how to handle that. Otherwise, I would definitely go and fix such label as I encounter them when doing changes. Addressing them at once could be hard. So if it is ok to make also these changes even though they are on different place than is PR about, I would change it then.

Reply:
This specific might require standalone pr, since also unit tests require updates...